src/Storage.Abstraction/ContainerClientBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Storage.Abstraction
+{
+    public class ContainerClientBase
+    {
+        
+        protected string ConnectionString 
+        { 
+            get {
+                if (this.config != null) return "conectionstring from config";
+                else throw new NotImplementedException(); 
+            }
+            set { }
+        }
+        private object config;
+        public ContainerClientBase(object config)
```

Comment:
Should not be better to use IOption or IConfiguration instead of object?

Reply:
I wanna have a config model! that should be a object. should be a concrete internal config model. I prefer not use IOption to not couple so much with .net framework ( about config topic i mean) . 

At tis point i let it so!